modules/command/dc/dc-offer-create-bc-command.js

```diff
@@ -46,13 +47,19 @@ class DCOfferCreateBcCommand extends Command {
             dataSizeInBytes,
             litigationIntervalInMinutes,
         );
+        const offerTaskEvent = await this.blockchain.decodeOfferTaskEventFromTransaction(result);
```

Comment:
What happens when this call fails? Here we are dependent on web3 service, is there any other service that we can use as a fallback?

Reply:
I don't think so. 
This is probably a static function, the bigger risk is if the transaction above returns an empty response, which does sometimes happen. 
I'm not sure how to handle failing to decode the data, I'll try to think of something. If you have any ideas I'd be glad to hear them